manim/container.py

```diff
@@ -0,0 +1,30 @@
+"""
+Abstract base class for several objects used by manim.  In particular, both
+:class:`~.Scene` and :class:`~.Mobject` inherit from Container.
+"""
+
+from abc import ABC, abstractmethod
+from .utils.config_ops import digest_config
+
+__all__ = ["Container"]
+
+
+class Container(ABC):
+    """Abstract base class for several manim objects.
+
+    Parameters
+    ----------
+    kwargs : Any
+        Arguments to be passed to :func:`digest_config`
+    """
+
+    def __init__(self, **kwargs):
+        digest_config(self, kwargs)
+
+    @abstractmethod
+    def add(self, *items):
+        """Abstract method to add items to Container."""
```

Comment:
Should prob document `items`, no? (Even if just as `Any`)

Reply:
done